docs/docs/draft/documentation/README.md

```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
 * `kumactl help [..]`: help dialog that explains the commands available.
 * `kumactl version [--detailed]`: shows the version of the program.
 
+## GUI
+
+Kuma now ships with a basic web-based GUI that will serve as a visual overview of your data planes, meshes, and various traffic policies.
+
+The GUI pairs with the HTTP API — Read more about the HTTP API here
```

Comment:
Let's add a link to HTTP API

Reply:
https://kuma.io/docs/latest/documentation/#http-api